src/watertap_contrib/reflo/costing/solar/pv_surrogate.py

```diff
@@ -0,0 +1,215 @@
+        bounds=(0, None),
+        doc="Cost per watt for permitting, environmental studies, engineering, land prep, and grid interconnection",
+    )
+
+    blk.land_cost_per_acre = pyo.Var(
+        initialize=4000,
+        units=costing.base_currency / pyo.units.acre,
+        bounds=(0, None),
+        doc="Land cost per acre required",
+    )
+
+    blk.contingency_frac_direct_capital_cost = pyo.Var(
+        initialize=0.03,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs for contingency",
+    )
+
+    blk.tax_frac_direct_capital_cost = pyo.Var(
+        initialize=0.05,
+        units=pyo.units.dimensionless,
+        bounds=(0, 1),
+        doc="Fraction of direct costs for sales tax",
+    )
+
+    blk.fixed_operating_by_capacity = pyo.Var(
+        initialize=0,
+        units=costing.base_currency / (pyo.units.kW * costing.base_period),
+        bounds=(0, None),
+        doc="Fixed operating cost of PV system per kW generated",
+    )
+
+    blk.variable_operating_by_generation = pyo.Var(
+        initialize=0,
+        units=costing.base_currency / (pyo.units.MWh * costing.base_period),
+        bounds=(0, None),
+        doc="Annual operating cost of PV system per MWh generated",
+    )
+
+    # blk.fix_all_vars()
+
+
+@register_costing_parameter_block(
+    build_rule=build_pv_surrogate_cost_param_block, parameter_block_name="pv_surrogate"
+)
+def cost_pv_surrogate(blk):
+
+    global_params = blk.costing_package
+    pv_params = blk.costing_package.pv_surrogate
+    make_capital_cost_var(blk)
+    make_variable_operating_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+
+    blk.direct_capital_cost = pyo.Var(
+        initialize=0,
+        units=blk.config.flowsheet_costing_block.base_currency,
+        bounds=(0, None),
+        doc="Direct costs of PV system",
+    )
+
+    blk.indirect_capital_cost = pyo.Var(
+        initialize=0,
+        units=blk.config.flowsheet_costing_block.base_currency,
+        bounds=(0, None),
+        doc="Indirect costs of PV system",
+    )
+
+    # blk.direct_capital_cost = pyo.Var(
+    #     initialize=0,
+    #     units=blk.config.flowsheet_costing_block.base_currency,
+    #     bounds=(0, None),
+    #     doc="Direct costs of PV system",
+    # )
+
+    # blk.indirect_capital_cost = pyo.Var(
+    #     initialize=0,
+    #     units=blk.config.flowsheet_costing_block.base_currency,
+    #     bounds=(0, None),
+    #     doc="Indirect costs of PV system",
+    # )
+
+    blk.land_cost = pyo.Var(
+        initialize=0,
+        units=blk.config.flowsheet_costing_block.base_currency,
+        bounds=(0, None),
+        doc="Land costs of PV system",
+    )
+
+    blk.sales_tax = pyo.Var(
+        initialize=0,
+        units=blk.config.flowsheet_costing_block.base_currency,
+        bounds=(0, None),
+        doc="Sales tax for PV system",
+    )
+
+    blk.system_capacity = pyo.Var(
```

Comment:
is this and `annual_generation` defined anywhere?

Reply:
Removed the unused costing variables